server/gokbg3/grails-app/controllers/gokbg3/UrlMappings.groovy

```diff
@@ -19,6 +19,7 @@ class UrlMappings {
       put "/packages/$id/tipps"(controller: 'package', namespace: 'rest', action: 'updateTipps')
       patch "/packages/$id/tipps"(controller: 'package', namespace: 'rest', action: 'updateTipps')
       "/packages/$id/$action"(controller: 'package', namespace: 'rest')
+      get "/packages/$id/jobs/"(controller: 'package', namespace: 'rest', action: 'jobs')
```

Comment:
Not really needed, since the line above already dynamically calls the method `jobs`.